[2024-07-22T16:57:07.126+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-22T16:57:07.178+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.merge_data scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-07-22T16:57:07.194+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.merge_data scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-07-22T16:57:07.195+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-22T16:57:07.224+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): merge_data> on 2024-07-16 00:00:00+00:00
[2024-07-22T16:57:07.273+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-22T16:57:07.286+0000] {standard_task_runner.py:63} INFO - Started process 205 to run task
[2024-07-22T16:57:07.332+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'merge_data', 'scheduled__2024-07-16T00:00:00+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpf9eeaa3_']
[2024-07-22T16:57:07.334+0000] {standard_task_runner.py:91} INFO - Job 72: Subtask merge_data
[2024-07-22T16:57:07.629+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline.merge_data scheduled__2024-07-16T00:00:00+00:00 [running]> on host ada6cb1e64b1
[2024-07-22T16:57:08.015+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='merge_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-16T00:00:00+00:00'
[2024-07-22T16:57:08.018+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-22T16:57:48.854+0000] {demo.py:60} INFO - Data Merged
[2024-07-22T16:57:49.504+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-22T16:57:49.514+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-22T16:57:49.569+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=merge_data, run_id=scheduled__2024-07-16T00:00:00+00:00, execution_date=20240716T000000, start_date=20240722T165707, end_date=20240722T165749
[2024-07-22T16:57:49.681+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-22T16:57:49.746+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-22T16:57:49.749+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
